salt-mode.el

```diff
@@ -82,6 +82,50 @@ suitable for spellchecking."
 
 (put 'salt-mode 'flyspell-mode-predicate #'salt--flyspell-predicate)
 
+(defun salt--state-module-at-point ()
```

Comment:
Good point. I'm not concerned with the BC break at this stage, so have fixed this in the master branch in https://github.com/glynnforrest/salt-mode/commit/c5ff43cdfbca5ccd8f212f575235eedafaa091ac, which you can merge into your eldoc branch.